src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -133,6 +133,12 @@ case class AgoraEntity(namespace: Option[String] = None,
                        managers: Seq[String] = Seq(),
                        public: Option[Boolean] = None) {
 
+  lazy val entityAlias: String =
+    namespace.get + "." + name.get + "." + snapshotId.get
```

Comment:
`namespace.get` can be replaced by `namespaceAlias`

Reply:
That does work but I like the symmetry between the two functions and not having them rely on one another (not a strong opinion)

Reply:
If you do decide to leave it, how about a comment so someone doesn't come behind you and change it without seeing this PR?

Reply:
I will accept the majority opinion in that case 🙂